.babelrc

```diff
@@ -26,25 +26,47 @@
       ]
     },
     "test": {
-      "presets": [ "es2015-node4", "stage-2", "stage-3", "flow", "react"],
+      "presets": [["env", {
+        "targets": {
+          "node": "6.10"
+        }
+      }], "flow", "react"],
+      "plugins": [
+        "transform-class-properties",
+        "transform-object-rest-spread"
+      ]
+    },
+    "testE2E": {
+      /** for an issue on puppeteer env we need to use es2015-node4 **/
```

Comment:
No issue with puppeteer, I did not manage to set up `babel-preset-env` for such environment. We use `await` with `NodeEnvironment` and for some reason, there is an issue with `jest-environment-node` and transpiling. I did not want to spend much time on it since we will move soon to Babel7.